libnopegl/src/backends/vk/gpu_ctx_vk.c

```diff
@@ -1391,6 +1391,19 @@ static int vk_get_preferred_depth_stencil_format(struct gpu_ctx *s)
     return vk->preferred_depth_stencil_format;
 }
 
+static uint32_t vk_get_format_features(struct gpu_ctx *s, int format)
```

Comment:
`const` for `s`?

Reply:
`ngli_gpu_ctx_get_format_features()` is not `const`. I believe there is a few other place where we can make the context const but since this internal API is moving I would rather do that once it is more mature (and avoid changing, gpu_ctx + gpu_ctx_gl + gpu_ctx_vk back and forth).